src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Dynamic Sampling
+sidebar_order: 10
+description: "Learn about the different methods for controlling the amount and type of data sampled for your project."
+---
+
+<Note>
+
+This content is provided on a working branch as part of a project that is in progress. We anticipate it will be changed and modified as the project progresses.
+
+</Note>
+
+Dynamic sampling addresses the need to easily change and manage what is sampled after an application is deployed. In addition, dynamic sampling enables more sophisticated sampling decisions that span multiple transactions.
+
+When you select your sampling type, you'll find that dynamic sampling:
+
+- prioritizes impactful data
+- enables observation of complete traces
+- maintains a representative sample
+- requires minimal configuration and no redeployment
+
+## Sampling Types
+
+Sentry offers the ability to sample in two different ways. You can sample [client-side in your SDK](/platform-redirect/?next=/configuration/sampling/) during SDK initialization or by [setting dynamic sampling rates](TODO: add link to new content) in Filters & Sampling for your Project in sentry.io. Learn more about our [sampling types and the benefits of each](/product/data-management-settings/dynamic-sampling/sampling-types/).
```

Comment:
"in Filters & Sampling for your Project in sentry.io" should we mention settings here?